src/policy.rs

```diff
@@ -70,7 +70,7 @@
 //! ```
 //! Example:
 //! If the current user of the sdk is "alice@ironcorelabs" and the following PolicyGrant is evaluated,
-//! `PolicyGrant::new("PII".try_from()?, "INTERNAL".try_from()?, None, None)` will match the second-to-last rule
+//! `PolicyGrant::new(Some(Category::try_from("PII")?), Some(Sensitivity::try_from("INTERNAL")?), None, None)` will match the second-to-last rule
```

Comment:
Do you think this would be better as a doc example? Then it would be compiled whenever we run the tests.
I'm fine either way, but if you wanted to add it, it would be something like this: 
```
//! ```
//! # use ironoxide::prelude::*;
//! # fn run() -> Result<(), IronOxideErr> {
//! # use std::convert::TryInto;
//! PolicyGrant::new(Some("PII".try_into()?), Some("INTERNAL".try_into()?), None, None);
//! # Ok(())
//! # }
//! ```
```

Reply:
Done, hopefully it turns out correctly in the docs.